blockchain/src/history/validity_store.rs

```diff
@@ -1,4 +1,55 @@
+use std::collections::{HashMap, HashSet};
+
+use nimiq_block::Block;
+use nimiq_hash::Blake2bHash;
+use nimiq_transaction::historic_transaction::RawTransactionHash;
+
 /// The validity store is used by full nodes to keep track of which
 /// transactions have occurred within the validity window without
 /// having to store the full transactions
-pub struct ValidityStore {}
+pub struct ValidityStore {
+    /// The set of all raw transactions of the current validity window.
+    pub(crate) _txs: HashSet<Blake2bHash>,
+
+    pub(crate) _block_txs: HashMap<u32, HashSet<Blake2bHash>>,
```

Comment:
```suggestion
    pub(crate) block_txs: HashMap<u32, HashSet<Blake2bHash>>,
```

Reply:
Missing rustdoc and why is it underscored?